src/main/kotlin/no/nav/syfo/personstatus/PersonoversiktStatusService.kt

```diff
@@ -57,134 +42,28 @@ class PersonoversiktStatusService(
     fun getPersonstatus(personident: PersonIdent): PersonOversiktStatus? =
         personoversiktStatusRepository.getPersonOversiktStatus(personident)
 
-    suspend fun getAktiveVurderinger(
+    suspend fun getAktiveOppgaver(
```

Comment:
Rename denne og kaller videre på ny service for å hente aktive oppgaver. 
Fortsatt litt rart at denne både henter aktive oppgaver og mapper til DTO så vi kunne evt flyttet dette ut til api-laget :thinking: